modules/core/src/main/java/org/apache/ignite/internal/util/StripedExecutor.java

```diff
@@ -139,20 +143,28 @@ public StripedExecutor(
     }
 
     /**
-     * Checks starvation in striped pool. Maybe too verbose
-     * but this is needed to faster debug possible issues.
+     * Checks starvation in striped pool.
      */
     public void checkStarvation() {
```

Comment:
Why do we need to create new method? Note this is an internal package, so we can just change return type.

Reply:
I wanted it to be shown via JMX, it seems more convenient for a user to see a result without watching logs, for instance if he disabled automatic check. But I was afraid of removing the old method as it's also JMX-exposed.

Reply:
@alamar you mean that I shouldn't have added new MX method in addition to the old one?

Reply:
It's OK that you have added new method to MX bean, but you should reuse existing method in StripedExecutor class, or remove it outright when adding a new one.